tests/test_config.py

```diff
@@ -233,8 +233,8 @@ def test_apply_defaults_for_has_value(self):
         assert 'dk_ncores' not in self.ucfg.cfg['precip'].keys()
 
 
-class TestMasterConfig(unittest.TestCase):
-    def setUp(self):
+class TestMasterConfig():
+    def setup_method(self):
```

Comment:
Think it would be nice to be consistent with new lines between class declaration and the first method.